package-lock.json

```diff
@@ -0,0 +1,12963 @@
+{
```

Comment:
Since our repo now uses Yarn, we should probably remove this `package-lock.json` file.